src/commands/convert/index.ts

```diff
@@ -137,6 +138,12 @@ export default class Convert extends Command {
         break
       }
 
+      case 'msft_secure_score': {
+        converter = new MsftSecureScoreMapper(fs.readFileSync(flags.input, 'utf8'))
+        fs.writeFileSync(checkSuffix(flags.output), JSON.stringify(converter.toHdf()))
```

Comment:
we're trying to ensure that output is formatted on write now.  for reference please see @kemley76's latest pr.